lib/u3d/installer.rb

```diff
@@ -169,9 +184,12 @@ def uninstall(unity: nil)
     private
 
     def list_installed_paths
-      find = File.join(DEFAULT_MAC_INSTALL, 'Applications', 'Unity*', 'Unity.app')
-      paths = Dir[find]
-      paths = paths.map { |u| Pathname.new(u).parent.to_s }
+      paths = find_installations_with_path(
+        DEFAULT_MAC_INSTALL,
```

Comment:
Side note: I made some work on #287 that was changing this. In particular I was modifying the `DEFAULT_MAC_INSTALL` on Mac to be `/Applications`. To make it more similar to the way the other platforms are managed.

Reply:
We can deal with this later. It was just a reminder.

Reply:
Yep, I saw your work because I got a bit startled at the Applications being repeated all the way through, but I'm not sure that this would work. Unity installation path takes a volume as an argument. I tried doing `u3d install 2019.2.1.f1 --installation_path ~` and it wouldn't work.